examples/Vanilla/Conversion/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html class="no-js" lang="">
```

Comment:
Remove "no-js" since we are not using Modernizr

Reply:
OK